sentry-core/src/main/java/io/sentry/core/SentryClient.java

```diff
@@ -79,7 +79,17 @@ public SentryClient(final @NotNull SentryOptions options, @Nullable Connection c
     }
 
     for (EventProcessor processor : options.getEventProcessors()) {
-      event = processor.process(event, hint);
+      try {
```

Comment:
M: Since this is not duplicated (on a difference list of event processors), could this be extracted to a private method that takes `List<EventProcessor>` and `SentryEvent` and called from here and line 288?

Reply:
nice catch